common/browser_options.go

```diff
@@ -127,15 +129,42 @@ func (l *LaunchOptions) Parse(ctx context.Context, logger *log.Logger, opts goja
 	return nil
 }
 
-func (l *LaunchOptions) shouldIgnoreOnCloud(opt string) bool {
-	if !l.onCloud {
-		return false
+func (l *LaunchOptions) shouldIgnoreOpt(opt string) bool {
```

Comment:
I think we only need `shouldIgnoreIfBrowserIsRemote`, those options will also be irrelevant in the "other" environment.

Reply:
> The one suggestion (which might be too late now) is that it could have been a good opportunity to move `browserprocess` into its own package like we tried [here](https://github.com/grafana/xk6-browser/pull/755).

@grafana/k6-browser 

I wouldn't do that in this PR as it's unrelated to its purpose. Another PR is welcome if we export only a few fields, unlike before, and develop a nicer solution (no pun intended! :)).

Reply:
But you say this based on the preliminary proposed solution for running browsers in the cloud right? Should we assume already that the browser is gonna be remote and therefore apply the changes?

Reply:
Also based on what options have been ignored in the remote setup, should also be ignored in the cloud, no?

Reply:
Remote/cloud is the same thing, at least for now. If we want to let users connect to remote browsers, that's a different story, and then we would need to separate cloud/remote run meanings. I remember a few users asked for this feature once.

Reply:
I would also be in favour of removing cloud and keeping remote, as we should refrain from using `cloud`.